test/unit/init/CliRunner.test.ts

```diff
@@ -0,0 +1,40 @@
+import * as Path from 'path';
+import type { Loader } from 'componentsjs';
+import { runCli } from '../../../src/init/CliRunner';
+import type { Setup } from '../../../src/init/Setup';
+
+let calledInstantiateFromUrl: boolean;
+let calledRegisterAvailableModuleResources: boolean;
+
+const mockSetup = {
+  setup: jest.fn(),
+} as unknown as jest.Mocked<Setup>;
+
+// eslint-disable-next-line arrow-body-style
+jest.mock('componentsjs', (): any => {
+  return function(): Loader {
+    return {
+      instantiateFromUrl(): any {
+        calledInstantiateFromUrl = true;
+        return mockSetup;
+      },
+      registerAvailableModuleResources(): any {
+        calledRegisterAvailableModuleResources = true;
+      },
+    } as unknown as Loader;
+  };
+});
```

Comment:
The following solves it. Reason being that Loader is not a default export of `componentsjs` but a named export. So the mock should return an object where the Loader key contains the mocked class.

This doesn't fix the tests though. First reason being that the logging related calls will error and will also need a mock, second one being that the CliRunner function creates a promise but doesn't wait for it to execute. So you'll probably have to add something in mock that you can await before doing the expect checks.

```suggestion
jest.mock('componentsjs', (): any => (
  // eslint-disable-next-line @typescript-eslint/naming-convention, object-shorthand
  { Loader: function(): Loader {
    return {
      instantiateFromUrl(): any {
        calledInstantiateFromUrl = true;
        return mockSetup;
      },
      registerAvailableModuleResources(): any {
        calledRegisterAvailableModuleResources = true;
      },
    } as unknown as Loader;
  } }
));
```